Assets/Scripts/KnitbyController.cs

```diff
@@ -0,0 +1,103 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+/// <summary>
+/// Updates Knitby's position to follow the player's path
+/// </summary>
+public class KnitbyController : MonoBehaviour
+{ 
+    [Header("References")]
+    [SerializeField] private LineRenderer lineRenderer;
```

Comment:
This is forcing an override in Knitby- we should get the LineRenderer directly from the player during Awake instead.

Also, future task for animating: the swing line should be animated to appear from Knitby and then attach to the player. When we make that change the LineRenderer also should no longer be controlled from the PlayerController.

Reply:
going to change it when we change line renderer

Reply:
changed